honeycomb-core/src/cmap/builder/grid/mod.rs

```diff
@@ -72,7 +72,7 @@ impl<T: CoordsFloat> CMapBuilder<T> {
         let gridd = GridDescriptor::default()
             .n_cells([n_square; 3])
             .len_per_cell([T::one(); 3]);
-        CMapBuilder::from_grid_descriptor(gridd)
+        CMapBuilder::from(gridd)
```

Comment:
Can we use `into()` here? We already know that we will return a `CMapBuilder`.

Reply:
Yes we can, nice catch